bindings/pyroot_experimental/PyROOT/src/PyzPythonHelpers.cxx

```diff
@@ -19,12 +19,116 @@ PyROOT extension module.
 
 #include "CPyCppyy.h"
 #include "CPPInstance.h"
+#include "ProxyWrappers.h"
 #include "PyROOTPythonize.h"
 #include "RConfig.h"
 #include "TInterpreter.h"
 
 #include <sstream>
 
+////////////////////////////////////////////////////////////////////////////
+/// \brief Get size of C++ data-type
+/// \param[in] obj PyObject with array interface
+///
+/// This function returns an RVec which adopts the memory of the given
+/// PyObject. The RVec takes the data pointer and the size from the array
+/// interface dictionary.
+PyObject *PyROOT::AsRVec(PyObject * /*self*/, PyObject * obj)
+{
+   if (!obj) {
+      PyErr_SetString(PyExc_RuntimeError, "Invalid Python object.");
+      return NULL;
+   }
+
+   // Get array interface of object
+   auto pyinterface = PyObject_GetAttrString(obj, "__array_interface__");
+   if (!pyinterface) {
+      PyErr_SetString(PyExc_RuntimeError, "__array_interface__ does not exist.");
```

Comment:
Can we precede the error messages (this one and the ones below) with something like "Object not convertible: "